osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/configuration/service/ProtocolServiceLookupTest.java

```diff
@@ -0,0 +1,48 @@
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.configuration.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.Protocol;
+import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ProtocolAdapterException;
+
+public class ProtocolServiceLookupTest {
```

Comment:
this test is unnecessary because the Spring Framework has built in lookup of beans at runtime.

Reply:
This test is only unnecessary if I remove the ProtocolServiceLookup, which is not the case.

Actually this is more evidence that having this lookup is a Good Idea. If the beans were hard wired by name it would be much harder to test the lookup logic.